tests/raftstore/server.rs

```diff
@@ -112,16 +116,16 @@ impl Simulator for ServerCluster {
         cfg.addr = format!("{}", addr);
 
         let simulate_trans = SimulateTransport::new(strategy, trans.clone());
-        let mut node = Node::new(&cfg,
-                                 self.pd_client.clone(),
-                                 Arc::new(RwLock::new(simulate_trans)));
+        let trans_final = Arc::new(RwLock::new(simulate_trans));
```

Comment:
confuse. 
I though final is the latest transport before. should think a better name.